environments/contrail/install_vrouter_kmod.yaml

```diff
@@ -397,11 +398,11 @@ resources:
                   ip route add default via $def_gw
                 fi
               fi
-              if [[ ${sriov_on} == 'true' ]]; then
+              if [[ ${sriov_on} == 'True' ]]; then
                 sed 's/^\(GRUB_CMDLINE_LINUX=".*\)"/\1 $KERNEL_ARGS intel_iommu=on iommu=pt"/g' -i /etc/default/grub ;
                 grub2-mkconfig -o /etc/grub2.cfg
                 systemctl enable supervisor-vrouter
```

Comment:
remove 'systemctl enable supervisor-vrouter' , because actually here the agent package is not installed so there is no such service.